perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -217,6 +220,9 @@ export default {
     isProductOwner() {
       return this.product.userData.canEdit || this.product.userData.username === this.product.creator.id || this.product.userData.username === this.product.receiverMarchand.id ;
     },
+    isProductMerchant() {
+      return this.product.receiverMarchand.id === eXo.env.portal.userName;
```

Comment:
I am not sure this is needed as we can rely just on the computed `isProductOwner`

Reply:
There is a difference between the creator of the product and the merchant of the product, when the merchant has been changed, we have to disable the commands of the menu icon for the creator and enable them for the new merchant.